pyiron/atomistics/structure/atoms.py

```diff
@@ -227,6 +228,7 @@ def __init__(
             else:
                 self.pbc = pbc
         self.set_initial_magnetic_moments(magmoms)
+        self._high_symmetry_points = high_symmetry_points
```

Comment:
I think it might be better to do something like

```
if high_symmetry_points is not None:
   self.set_high_symmetry_points(high_symmetry_points)
```
This way, it will nicely raise an error if this parameter is not a `dict`

Reply:
done